amsclient/amsclient.go

```diff
@@ -242,6 +242,8 @@ func (c *amsClientImpl) executeSubscriptionListRequest(
 	clusterInfoList []types.ClusterInfo,
 	err error,
 ) {
+	uniqueClusterMap := make(map[string]bool)
```

Comment:
you can use a map of empty structs which is optimized (or should be according to Go spec/idioms).

Reply:
done